go.mod

```diff
@@ -37,7 +37,7 @@ require (
 	google.golang.org/protobuf v1.28.1 // indirect
 	gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
-	k8s.io/klog/v2 v2.70.0 // indirect
+	k8s.io/klog v1.0.0 // indirect
```

Comment:
This leads me to think that we are still using klog v1 somewhere, or is that related to klog v2 having a relationship with klog v1?